server/game/cards/06-WoE/ShrewdInvestor.js

```diff
@@ -0,0 +1,23 @@
+const Card = require('../../Card.js');
+
+class ShrewdInvestor extends Card {
+    // Play: You may have your opponent gain 1Aember. If you do,
+    // capture 4Aember.
+    setupCardAbilities(ability) {
+        this.play({
+            optional: true,
+            gameAction: ability.actions.sequential([
+                ability.actions.gainAmber((context) => ({
+                    target: context.player.opponent
+                })),
+                ability.actions.capture({ amount: 4 })
+            ]),
+            message: '{0} uses {1} to cause {2} to gain 1 amber, and then capture 4 amber onto {0}',
```

Comment:
I think you should remove this `onto {0}` as zero is the player's name

Reply:
Yep, fixed.